environment/src/test/java/com/sequenceiq/environment/environment/encryption/EnvironmentEncryptionServiceTest.java

```diff
@@ -204,19 +206,22 @@ void testCreateEncryptionResourcesDeletionRequestShouldReturnWithANewDeletionReq
                 .withParameters(ParametersDto.builder()
                         .withAzureParameters(AzureParametersDto.builder()
                                 .withEncryptionParameters(AzureResourceEncryptionParametersDto.builder()
-                                        .withDiskEncryptionSetId(DISK_ENCRYPTION_SET_ID).build())
+                                        .withDiskEncryptionSetId(DISK_ENCRYPTION_SET_ID)
+                                        .withEncryptionKeyUrl(KEY_URL)
+                                        .build())
                                 .build())
                         .build())
                 .withCreator(USER_NAME)
                 .withAccountId(ACCOUNT_ID)
                 .build();
         CloudResource desCloudResource = CloudResource.builder()
-                .name("Des")
+                .name(DISK_ENCRYPTION_SET_NAME)
                 .type(ResourceType.AZURE_DISK_ENCRYPTION_SET)
                 .reference(DISK_ENCRYPTION_SET_ID)
                 .status(CommonStatus.CREATED)
                 .build();
-        when(resourceRetriever.findByResourceReferenceAndStatusAndType(any(), any(), any())).thenReturn(Optional.ofNullable(desCloudResource));
+        when(resourceRetriever.findByEnvironmentIdAndType(ENVIRONMENT_ID, ResourceType.AZURE_DISK_ENCRYPTION_SET))
+                .thenReturn(Optional.ofNullable(desCloudResource));
```

Comment:
Nit: `desCloudResource` cannot be `null` here, so `Optional.of()` would be more appropriate.